panel/configuration.md

```diff
@@ -16,15 +47,15 @@ highly suggest providing a specific IP address (or comma seperated list of IPs)
 if your proxy is running on the same machine as the server, chances are that something like `TRUSTED_PROXIES=127.0.0.1`
 will work for you.
 
-### Cloudflare Specific Configuration
+#### Cloudflare Specific Configuration
 If you're using Cloudflares Flexible SSL you should set `TRUSTED_PROXIES` to contain [their IP addresses](https://www.cloudflare.com/ips/).
 Below is an example of how to set this.
 
 ```
 TRUSTED_PROXIES=103.21.244.0/22,103.22.200.0/22,103.31.4.0/22,104.16.0.0/12,108.162.192.0/18,131.0.72.0/22,141.101.64.0/18,162.158.0.0/15,172.64.0.0/13,173.245.48.0/20,188.114.96.0/20,190.93.240.0/20,197.234.240.0/22,198.41.128.0/17
 ```
 
-## Increasing Editable File Size
+### FILES_MAX_EDIT_SIZE - Increasing Editable File Size
```

Comment:
Please remove the environment variable names from the section titles, looks weird.